src/utils/hook.ts

```diff
@@ -540,10 +525,25 @@ export function useChartQueryWithCache<T>(
       dataList = dataList.data.map(wrapper => wrapper.attributes)
     }
     if (cacheKey && dataList.length > 0) {
-      const storeCache =
-        // eslint-disable-next-line no-nested-ternary
-        cacheMode === 'forever' ? storeCachedData : cacheMode === 'date' ? storeDateChartCache : storeEpochChartCache
-      storeCache<T[]>(cacheKey, dataList)
+      let expireAt: Date | number
+      switch (cacheMode) {
+        case 'epoch':
+          expireAt = Date.now() + ONE_HOUR_MILLISECOND * 3
+          break
+        case 'date': {
+          // Chart data will be updated at 0:10(CST) every day
```

Comment:
@rabbitz the timing of updating charts has been changed to 8:00(CST), am I right?

Reply:
The calculation for DailyStatistic currently takes place at 8:00 AM (CST). However, it may take some time to complete. The cache update can be delayed by a few minutes, for example, until 8:05 AM (CST).

Reply:
LGTM

Reply:
done